pkg/codegen/importer/testdata/cases.json

```diff
@@ -588,16 +588,20 @@
 						"targetGroupArn": "foobar",
 						"type": "foobar"
 					}
-				],
-				"conditions": [
-					{
-						"field": "foobar",
-						"values": "foobar"
-					},
-					{
-						"field": "foobar",
-						"values": "foobar"
-					}
+                ],
+                "conditions": [
+                    {
+                        "queryStrings": [
+                            {
+                                "key": "foobar",
+                                "value": "foobar"
+                            },
+                            {
+                                "key": "foobar",
+                                "value": "foobar"
+                            }           
+                        ]
+                    }
```

Comment:
Is the indentation off here? The diff looks odd but I can't quite tell if this is intentional.

Reply:
It's not intentional. `cases.json` does indentation vie tabs, and I put spaces. It was right for the size of tabs as displayed on my editor, but wrong for GitHub. The next push will be all tabs.